shell/common/gin_converters/guid_converter.h

```diff
@@ -34,6 +38,9 @@ struct Converter<UUID> {
     UUID uid;
 
     if (guid.length() > 0) {
+      if (guid[0] == '{' && guid[guid.length() - 1] == '}') {
```

Comment:
any particular reason to support these extra optional curlies? should we just require that the raw uuid be passed in?

Reply:
Its very common to have the curly brackets on Windows. Our own `base::win::WStringFromGUID` method adds them. So for consistency, I added support to read them with our without.